Source/Bogus/DataSets/Date.cs

```diff
@@ -33,14 +43,16 @@ public Date(string locale = "en") : base(locale)
          this.hasWeekdayAbbrContext = HasKey("weekday.abbr_context", false);
       }
 
+      protected internal DateTime GetTimeReference() => LocalSystemClock?.Invoke() ?? SystemClock();
```

Comment:
Do you want to expose that method or now? Since we have Recent and Future, wouldn't it make sense to also expose a "Now"? Or would that be confusing since its the reference time?

Reply:
IMO it should not be exposed unless there is a very good reason for it.

(because of maintenance)

Reply:
hm.. very interesting proposal @garcipat for `.Now()`; conceptually it does make sense -- I'll think about it;

the only concern I would probably raise is consistency if we were to introduce `.Now()` would probably be the different types of "now". eg: `DateTime` now, `DateTimeOffset` now, `DateOnly` now, `TimeOnly` now variants of "now" if we were being consistent with our APIs.